checkov/terraform/runner.py

```diff
@@ -216,6 +202,26 @@ def run(
 
         return report
 
+    def update_definitions_and_breadcrumbs(self, all_graphs, local_graph, report, root_folder):
```

Comment:
should probably be private function to make sure it's more clear it's an inner function